### PR TITLE
fix(config): add metadata to Leviton devices and improve param definitions

### PR DESCRIPTION
**Pull request by disforw**

### CODE REVIEW

**Comment by disforw** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
Your file has syntax errors, specifically the closing ] after options.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
There are still syntax errors. Please use a JSON validator to correct them.

File: packages/config/config/devices/0x001d/dz6hd.json [hunk 1]
```diff
@@ -50,30 +62,78 @@
...
 			"label": "LED Dim Level Indicator Timeout",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 255,
 			"defaultValue": 3,
-			"unsigned": true
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Level Indicators OFF",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/dz6hd.json]:
```suggestion
					"label": "Always off",
```

File: packages/config/config/devices/0x001d/dz6hd.json [hunk 2]
```diff
@@ -50,30 +62,78 @@
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Memory Dim",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/dz6hd.json]:
```suggestion
					"label": "Last non-zero level",
```
Is probably a bit clearer (if I understand this correctly?)

File: packages/config/config/devices/0x001d/dz6hd.json [hunk 3]
```diff
@@ -50,30 +62,78 @@
...
+					"value": 0
+				}
+			]
 		},
 		"6": {
 			"label": "LED Dim Level Indicator Timeout",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/dz6hd.json:79]: I think this is missing the unit and the "Always on" (255) option.

File: packages/config/config/devices/0x001d/dz6hd.json [hunk 4]
```diff
@@ -21,15 +21,27 @@
...
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Instant off",
+					"value": 0
+				}
+			]
 		},
 		"3": {
 			"label": "Minimum Light Level",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/dz6hd.json:49]: Z-Wave levels usually go from 0 to 99.
Can you confirm that these actually go from 0 to 100 - so an actual percentage?
In this case we should add `"unit": "%"` to params 3 .. 5

File: packages/config/config/devices/0x001d/dz6hd.json [hunk 5]
```diff
@@ -21,15 +21,27 @@
 			"minValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x001d/dz6hd.json:22]: This and the following parameter should have an explanation of the values in the description:
```
Values 1-127 = seconds; 128-253 = minutes (minus 127)
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, a few minor changes please.

File: packages/config/config/devices/0x001d/dz6hd.json (hunk 3 above)
Comment (disforw) [reply packages/config/config/devices/0x001d/dz6hd.json:79]: I missed it, adding now…

**Comment by disforw** [comment]
@zwave-js-bot fix lint

**Comment by disforw** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**